plugin-server/src/utils/db/hub.ts

```diff
@@ -194,31 +192,15 @@ export async function createHub(
         // an acknowledgement as for instance there are some jobs that are
         // chained, and if we do not manage to produce then the chain will be
         // broken.
-        try {
-            await kafkaProducer.producer.send({
-                topic: KAFKA_JOBS,
-                messages: [
-                    {
-                        key: job.pluginConfigTeam.toString(),
-                        value: JSON.stringify(job),
-                    },
-                ],
-            })
-        } catch (error) {
```

Comment:
~What's your reasoning around removing that try/catch? What's the new behaviour like?~ Nevermind, it's been moved to KafkaProducerWrapper instead